admin/main.js

```diff
@@ -325,19 +384,26 @@ $(document).on('submit', 'form.preset-form[data-table]', function () {
 });
 
 //add new preset
-$('#lxNew').click(function (){
+$('#lxNew').on('click', function(){
```

Comment:
Noice

Reply:
apparently .click is depreciated or something